### ISSUE TITLE
Issue uploading images

### ISSUE THREAD

**Issue by flaviozluca** [zulip/zulip-desktop#1272]
Friends,

Not sure if it happens in the web interface too, but I noticed, sometimes I upload an image in the app and it doesn't go through. I think Im inpatient and click elsewhere after it starts uploading, then it doesnt upload and I have to try again.

I will keep an eye out and see if it's what I believe that happens, as I wrote above. Not being patient enough to wait for the upload, avoid switching the window, etc.

![image](https://user-images.githubusercontent.com/44004645/214315225-c557ce0b-8256-4816-86e4-96c42efd02e5.png)

**Comment by eyahlin** [comment]
Yes, I've experienced this too. If you press "Enter" before the uploading ends, the filename of the linked image will remain at "Uploading...". My only option is to wait until the file fully uploads. 

It would be great if Zulip could change the filename to change the link to the actual image file after it successfully uploads, even if Enter was prematurely hit while uploading. This works well on Slack.

**Comment by Midas847** [comment]
@flaviozluca Could this be a network error from your side?

**Comment by flaviozluca** [comment]
Hi, friend

Thanks. It's exactly what Eyahlin replied that happens to her too...

See her interaction here, pls.

Thanks.

**Comment by stianjensen** [comment]
> Could this be a network error from your side?

There's no network error. Uploading a file simple takes a few seconds, and if you (by habit) press enter before it has finished, you get a broken link.

**Comment by alya** [comment]
Ah,  I see. We disable the "Send" button while uploading files, but don't disable sending the message via the keyboard.

**Comment by alya** [comment]
I think the most expedient fix is to disable sending via Enter / Ctrl + Enter while uploads are in progress, as we do with the mouse UI. We can improve on that later as a follow-up.
